tslint.json

```diff
@@ -29,11 +30,11 @@
             "static-before-instance",
             "variables-before-functions"
         ],
-        "no-any": true,
-        "no-arg": true,
+        "no-any": false,
+        "no-arg": false,
```

Comment:
no-arg: Why this one?  (link: https://palantir.github.io/tslint/rules/no-arg/ )

Reply:
Again, just trying to get the code to build